src/features/classification/AddClassificationBadge.scss

```diff
@@ -0,0 +1,19 @@
+@import '../../styles/variables';
+@import './mixins';
+
+.bdl-AddClassificationBadge {
+    @include bdl-ClassificationBadge;
+
+    background-color: $bdl-gray-02;
+    border-color: $bdl-gray-20;
+
+    &:hover {
+        background-color: $bdl-gray-05;
+    }
+}
+
+.bdl-AddClassificationBadge-name {
+    @include bdl-ClassificationBadge-name;
+
+    color: $bdl-gray-50 !important; // To inhibit overriding
```

Comment:
We've managed to avoid this in most cases. What would override it?

Reply:
Very unfortunate, there is code both in EUA and here that adds styling on direct tags (h1 etc) and on * that I couldn't fix without a much bigger refactor or without duplicating styles that are here and then again in EUA. Line 34 here https://github.com/box/box-ui-elements/blob/master/src/components/breadcrumb/Breadcrumb.scss#L34

This was the least offensive way for now.